src/providers/Project/projectMutator.ts

```diff
@@ -106,8 +124,8 @@ export default class ProjectMutator {
     } else {
       Mixpanel.track('Project saved', { projectId: this.projectId });
     }
-    
-    navigate(`/${this.projectId}`, { replace: true});
+
+    navigate(`/${this.projectId}?type=readme&id=0`, { replace: true });
```

Comment:
I think the README should just be the default view when loading a project. Is `?type=readme` required?